tutorials/aptos-tutorial.md

```diff
@@ -0,0 +1,269 @@
+
+After you install it, go to File -> Settings, select `Plugins` on the left, then `Marketplace` and 
+search for the `Move language`.      
+
+### Creating new project
+
+Move projects are called Packages, and contain multiple Modules. 
+
+Module is a smart-contract that combine types and functions, and provides a very unique set of rules and restrictions 
+to manage relationships between them.  
+
+Package is a set of related Move modules which share common dependencies and often published
+to the same address in the Aptos blockchain.
+
+First, create a directory and initialize it with a Package using aptos-cli:
+
+```shell
+mkdir userinfo
+cd userinfo
+aptos move init --name UserInfo
+```
+This should be the resulting directory structure of our new Package:
+```
+userinfo/
+├── sources/
+└── Move.toml
+```
+
+`sources/` - directory is where you put your modules.  
+`Move.toml` - manifest file for the package. Here, you define package metadata, dependencies and addresses used in the code.
+
+Let's add an address with name `Sender` and value `0x42` under the `[addresses]`. We're going to store all our modules at that
+address in the Aptos blockchain. 
+
+```toml
+[package]
+name = "UserInfo"
+version = "0.0.0"
+
+[dependencies]
+AptosFramework = { git = "https://github.com/aptos-labs/aptos-core.git", subdir = "aptos-move/framework/aptos-framework/", rev = "main" }
+
+[addresses]
+Sender = "0x42"
+```
+
+`move init` command automatically adds a dependency on the `AptosFramework` package. It also transitively adds a `MoveStdlib`
+dependency. Let's compile that empty package to make Aptos CLI fetch the dependencies, and inspect them. 
+
+First time compilation could take a while, as it fetches the whole `aptos-core` repo from the internet.
+
+```shell
+~/userinfo $ ~/bin/aptos move compile
+{
+  "Result": []
+}
+```
+
+Fetched dependencies as well as build artifacts are stored in the `build/` directory at the package root. 
+
+```shell
+build/
+└── UserInfo
+    ├── bytecode_modules
+    ├── source_maps
+    └── sources
+        └── dependencies
+            ├── AptosFramework
+                ├── Account.move
+                ├── AccountUtils.move
+                ........
+            └── MoveStdlib
+                ├── ASCII.move
+                ├── Signer.move
+                ├── Vector.move
+                ........
+```
+
+`MoveStdlib` - standard library of the Move language, it consists of modules that are really indispensable, like functions 
+to work with vectors and signers. 
+
+`AptosFramework` - a set of modules specific to Aptos blockchain, like `Coin` module for ERC20-like, and `Account` 
+for the account metadata. 
+
+### Resources and storage
+
+TODO: talk about &signer and resources
+
+Every user of the Aptos blockchain has it's own object storage, located at the user address. There are builtin methods, 
+which allow to communicate with said storage from the Move code. 
```

Comment:
Maybe let's mention that arguments for that functions usually addresses, but to store something dev needs signer?